lib/index.js

```diff
@@ -184,7 +184,7 @@ Socket.prototype._receive = function () {
             this._stopPollSend();
             this._stopPollReceive();
             this.emit('survey-timeout');
-            return;
+            return null;
```

Comment:
returning `null` here inside the survey's `send` operation ends of the stream properly instead of getting `EOF`